drivers/adc-dac/ad74413r/ad74413r.c

```diff
@@ -338,13 +337,24 @@ int ad74413r_reset(struct ad74413r_desc *desc)
 {
 	int ret;
 
-	ret = ad74413r_reg_write(desc, AD74413R_CMD_KEY, AD74413R_CMD_KEY_RESET_1);
-	if (ret)
-		return ret;
+	if (desc->reset_gpio) {
```

Comment:
please update the function documentation since now it does not do only a "soft reset".